service/csv/pokemon.go

```diff
@@ -0,0 +1,60 @@
+package service_csv
+
+import (
+	"encoding/csv"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/raymundo/academy-go-q42021/model"
+)
+
+func GetPokemons() []model.Pokemon {
+	f, err := os.Open("assets/pokemons.csv")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer f.Close()
+
+	csvReader := csv.NewReader(f)
+	data, err := csvReader.ReadAll()
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	pokemons := formatPokemonList(data)
+	return pokemons
+}
+
+func formatPokemonList(data [][]string) []model.Pokemon {
+	var pokemons []model.Pokemon
+
+	for i, pokemon := range data {
+		var dataPokemon []string = pokemon
+		dataPokemon[1] = strings.ToLower(dataPokemon[1])
+
+		if i > 0 {
```

Comment:
I assume this is for the .csv file headers, if this is the case, you could instead use `data[1:]` instead of `data` in line 34, and get rid off of this if statement.